app/server/src/features/accounts/methods.ts

```diff
@@ -321,8 +289,12 @@ export async function makeOrganizer(prisma: PrismaClient, input: OrganizerForm,
     // TODO validate product key to elevate account to organizer
     // ensure viewer has privledge to make user organizer
     const queryResult = await prisma.user.findUnique({ where: { id: userId }, select: { canMakeOrgs: true } });
-    if (!queryResult) throw new Error(errors.DNE('user'));
-    if (!queryResult.canMakeOrgs) throw new Error(errors.permissions);
+    if (!queryResult)
+        throw new ProtectedError({
+            userMessage: 'Internal server error. Please try again later.',
+            internalMessage: errors.DNE('user'),
+        });
+    if (!queryResult.canMakeOrgs) throw new ProtectedError({ userMessage: errors.permissions });
```

Comment:
I think maybe consolidating errors into static fields on the ProtectedError class would make a lot of sense.